android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageView.kt

```diff
@@ -289,87 +274,25 @@ fun CreateListingPageView(
 					horizontalArrangement = Arrangement.SpaceBetween,
 					verticalAlignment = Alignment.CenterVertically,
 				) {
-					RoundedTextField(
-						modifier = Modifier
-							.fillMaxWidth(0.475f)
-							.border(
-								dimensionResource(id = R.dimen.xxxs),
-								textFieldBorderColor,
-								RoundedCornerShape(dimensionResource(id = R.dimen.xxxxl))
-							),
-						placeholder = {
-							Text(
-								text = stringResource(id = R.string.placeholder_date),
-								color = secondaryTextColor,
-							)
-						},
-						label = {
-							Text(
-								text = stringResource(id = R.string.start_date),
-								color = secondaryTextColor,
-							)
-						},
-						trailingIcon = {
-						   IconButton(
-							   onClick = {
-								   coroutineScope.launch {
-									   openBottomSheet = true
-								   }
-							   }
-						   ) {
-							   Icon(
-								   painter = painterResource(
-									   id = R.drawable.calendar_outline_gray_24
-								   ),
-								   contentDescription = stringResource(id = R.string.open_date_picker),
-							   )
-						   }
-						},
+					DateInputButton(
+						width = 0.475f,
```

Comment:
Hmm is there a good reason to do this? Seems like a weird UI choice, what does it look like / why are we doing this instead of using dynamic width + padding?

Reply:
good catch, made the change so it should be good now!